### DISCUSSION TITLE
Displays an error when I try to create a folder with mods in the mods directory!

### DISCUSSION THREAD

**Discussion by Delitel-WEB** [efroemling/ballistica#391] [Q&A] [answered]
When I try to create an EMPTY sys\1.6.4 folder, I get an error
![image](https://user-images.githubusercontent.com/57365921/169275467-400bbb4c-6011-4103-ae2e-4be45b2aec99.png)

**Comment by Dliwk** [comment, marked as answer]
You don't need to have this folder to install your mods. Just copy them to your 'mods' directory.

This `sys/<version>` is a special folder which should contain all important for game python modules, like `ba` and `bastd`. When this directory exists and is empty, game tries to find there (unsuccessfully) `ba` module and displays this error.

The only case when you want to use this folder is when you want to change very much things in internal modules. Or if you want to experiment with this directory, you can copy there all the files from 'ba_data/python'.
